swim.go

```diff
@@ -105,12 +105,11 @@ func (n *Node) Join(remote netip.AddrPort) {
 
 func (n *Node) send(ps []packet) {
 	for _, p := range ps {
-		dst, addrs := n.getAddrs(p)
-		if dst == (netip.AddrPort{}) {
+		if p.remoteAddr == (netip.AddrPort{}) {
```

Comment:
Under what circumstances is this possible?

Reply:
It shouldn't be; if it is, then there is a bug in the fsm or the way addresses are supplied to it. It seems like a more appropriate behavior in that case would be to panic. If so, then the subsequent call to WriteToUDPAddrPort would return an error, which accomplishes that, and this check can be removed.

Reply:
Sounds good to me! I agree.